.pre-commit-config.yaml

```diff
@@ -21,3 +21,8 @@ repos:
   hooks:
   - id: markdownlint
     args: ["--config", ".markdownlint.jsonc"]
+- repo: https://github.com/pre-commit/mirrors-prettier
+  rev: ''  # Use the sha / tag you want to point at
```

Comment:
What's the meaning of the emptry string?

Reply:
Oops... fixed! Thanks!